src/main/java/teammates/storage/entity/StudentProfile.java

```diff
@@ -170,4 +174,12 @@ public void setModifiedDate(Date modifiedDate) {
         this.modifiedDate = modifiedDate;
     }
 
+    // Equalizes Objectify and GenderType enum
+    public void importGender(@AlsoLoad("gender") String gender) {
```

Comment:
I'm not very familiar with Objectify. What does this annotation do here?

Reply:
@AlsoLoad("field") it is use to load the  field with the alternate name in the datastore . As in this I am using with the method having single parameter therefore this method will be called with the Datastore value